Code/Framework/AzToolsFramework/AzToolsFramework/API/EditorCameraBus.h

```diff
@@ -116,6 +135,16 @@ namespace Camera
          */
         virtual void ToggleCameraAsActiveView() = 0;
 
+        /**
+         * Aligns this camera with the active view in the scene and sets it as the active camera
+         */
+        virtual void MatchViewport() = 0;
+
+        /**
+         * Returns true if this is the active camera
+         */
+        virtual bool IsThisCamera() const = 0;
```

Comment:
suggestion: It might be worth just calling this `IsActiveCamera()` so the naming is clearer and more closely matches the comment. `IsThisCamera` seems like a new name which is less clear to the reader